deploy/helm/grimmory/templates/deployment.yaml

```
@@ -9,6 +15,34 @@ spec:
+    {{- $dbHost := "" }}
+    {{- $dbPort := 0 }}
+    {{- $dbName := "" }}
+    {{- $dbUserName := "" }}
+    {{- $dbPasswordSecretName := "" }}
+    {{- $dbPasswordSecretKey := "" }}
+    {{- if .Values.mariadb.enabled }}
+    {{- $dbHost = printf "%s-mariadb" .Release.Name  }}
+    {{- $dbPort = .Values.mariadb.service.port }}
+    {{- $dbName = .Values.mariadb.auth.database }}
+    {{- $dbUserName = .Values.mariadb.auth.username }}
+    {{- if .Values.mariadb.auth.existingSecret }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)